samples/build-duration-logger/README.md

```diff
@@ -4,18 +4,40 @@ In this JavaScript example, build durations are calculated and written out to th
 
 ## Concepts
```

Comment:
This “concepts” section is weird. Let's get rid of it.

Reply:
Done.

Reply:
Also renamed the examples.